templates/cloud/public-cloud.html

```diff
@@ -1,170 +1,313 @@
+          <div class="col-6 col-medium-3">
+            <p>
+              The provenance of base and application images is critical for security. While tooling like Docker makes retrieving and running images trivial, ensuring a trusted software supply chain is required. A simple way to get started is by using our hardened repositories. You can also work with us to get a custom built container with a 12 year security maintenance commitment.
+            </p>
+            <hr class="is-muted" />
+            <p>
+              <a href="/security/docker-images">Explore our container offering&nbsp;&rsaquo;</a>
+            </p>
           </div>
-          <div class="p-logo-section__item">
-            {{
-              image(
-                url="https://assets.ubuntu.com/v1/da9a1344-Microsoft-Azure-logo_stacked_transparent.png",
-                alt="Microsoft Azure",
-                width="350",
-                height="100",
-                hi_def=True,
-                attrs={"class": "p-logo-section__logo"},
-                loading="auto",
-              ) | safe
-            }}
+        </div>
+        <hr class="is-muted" />
+        <div class="row">
+          <div class="col-3 col-medium-3">
+            <h3 class="p-heading--5">Managed applications</h3>
           </div>
-          <div class="p-logo-section__item">
-            {{ image (
-              url="https://assets.ubuntu.com/v1/8c1c38b0-google-cloud-3.svg",
-              alt="Google Cloud",
-              width="350",
-              height="61",
-              hi_def=True,
-              attrs={"class": "p-logo-section__logo"},
-              loading="lazy"
-              ) | safe
-            }}
+          <div class="col-6 col-medium-3">
+            <p>
+              Accelerate innovation across clouds with fully managed Apache Kafka, Kubeflow, ElasticSearch and more. We build and operate the solution for you, backed by a full SLA. With Canonical's managed open source application services there's no lock in and you can take over control whenever you want.
+            </p>
+            <hr class="is-muted" />
+            <p>
+              <a href="/managed/apps">Learn more about Canonical Managed Apps&nbsp;&rsaquo;</a>
+            </p>
           </div>
         </div>
       </div>
     </div>
-  </div>
-</section>
+  </section>
 
-{% with first_item="_cloud_bootstack", second_item="_cloud_newsletter", third_item="_cloud_further_reading" %}{% include "shared/contextual_footers/_contextual_footer.html" %}{% endwith %}
+  <section class="p-section">
+    <div class="u-fixed-width">
+      <hr />
+    </div>
+    <div class="row">
+      <div class="col-3 col-medium-6">
+        <h2 class="p-muted-heading">
+          <a href="/blog/tag/public-cloud">Latest from our blog&nbsp;&rsaquo;</a>
+        </h2>
+      </div>
+      <div class="col-9 col-medium-6">
+        <div class="row">
+          {% for article in blogs %}
+            {% if loop.index <= 3 %}
```

Comment:
I could be wrong, but if we are pulling latest news there should always be at least 3 blogs

Reply:
So do you mean 3 or more, because if it is less than 3 then it would mean that the total blogs are less than 3, which is nearly impossible considering lots of tags that are being taken into account.

Reply:
I see, this should be fine then